### ISSUE TITLE
latest version of graphviz

### ISSUE THREAD

**Issue by EasyArray** [EasyArray/phosphorus#13]
Tested with the latest version of graphviz (v0.19) and trees fail. I think the `_repr_svg_` method of Graph objects went away? Could be replaced by something called `_repr_mimebundle_` ? Luckily Kaggle uses graphviz v0.8.4, so we should be good for now, but this could become an issue in the future.

**Comment by havijw** [comment]
I can add a check to make sure the version is correct when the package gets imported. Since we're still using Kaggle, it seems like a good thing to still use the old version, but I could also look into `_repr_mimebundle_` and maybe write a new version of `TreeVal._repr_svg_` but keep it commented out? That could be a project for another time though.

**Comment by EasyArray** [comment]
Yeah, I used the latest version of the installed program graphviz on Binder, and I used 0.8.4 of the python package interface. Keep it on the back burner to make our code resilient to multiple versions of graphviz (python package).
